app/components/elements/Checkbox.js

```diff
@@ -22,8 +22,8 @@ const StyledCheckbox = styled(Base)`
   &.checked {
     color: ${colors.purpleMedium};
     background-color: ${colors.white};
-    height: 1.5em;
-    width: 1.5em;
+    height: 1em;
```

Comment:
Quick question: do the width and height rules need to be specified here at all?  Would they not just be inherited by the ones above?

Reply:
It's something about the way rebass does it, they are actually separate svg elements, which in theory would be affected by the same parent element, but in practice I think there is something that's overruling it, so this was the best way I could find to handle it.